org.jacoco.ant.test/pom.xml

```diff
@@ -41,9 +41,18 @@
       <groupId>org.apache.ant</groupId>
       <artifactId>ant-antunit</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.ant</groupId>
+      <artifactId>ant-launcher</artifactId>
+    </dependency>
     <dependency>
       <groupId>org.apache.ant</groupId>
       <artifactId>ant-junit</artifactId>
+      <version>${ant.version}</version>
```

Comment:
Isn't this version already fixed in dependencyManagent of the parent POM?

Reply:
Good catch! Fixed